### DISCUSSION TITLE
Respiration data obtained from respiration belt: Bellows vs Cushion System?

### DISCUSSION THREAD

**Discussion by sebastian-kueppers** [bachlab/PsPM#789] [Q&A] [answered]
### Summary
Dear all,
in our experiment we obtain respiration data by using a respiration belt. This device is placed rather tightly on the upper abdomen of the participant and measures the abdomen's extraction when inhaling and contraction when exhaling. We are interested in obtaining a continuous measure of respiration rate, the inversion of the interval between breaths.
PsPM has two methods for breathing cycle detection: Bellows and Cushion system. I am rather new to physiological preprocessing and cannot tell which one is more suitable for our kind of respiration signal. I tried out both on some sample data and they give different results, e.g., for the average respiration rate over a 180sec baseline condition.

Do you have recommendations?

Thanks a lot in advance!

Best, Sebastian

### Technical Info
* PsPM revision ID: 6.1.2
* MATLAB version: R2023b

**Comment by dominikbach** [comment]
Hi Sebastian, could you give model/manufacturer of the device, and/or some more details on what it (physically) measures? Dominik

**Comment by sebastian-kueppers** [comment]
Hi Dominik, thanks for your reply!

We use Biosemi's Respiration belt SleepSense 1387-kit (https://www.biosemi.com/accessoires.htm): "_Using a piezo-electric crystal, the respiration effort sensor converts chest or abdominal respiration movement to clear, reliable respiration waveforms. Strong reliable signals are available because the respiration belt uses the powersupply from the ActiveTwo system. The respiration input can be mounted on the front of the AD-box when a system is ordered, or at a later stage._"

The piezo-electric crystal converts the mechanical stress resulting from the expansion and contraction of the chest/abdomen when breathing into an electrical current. The result is a wavy, almost sine-like signal.

**Comment by dominikbach** [comment, marked as answer]
I haven't seen waveforms from this system, so let me summarise the main features of the implemented routines:
- bellows system: increase at start of inspiration, peak at maximum inspiration, but returns to zero when inspiration is held
- cushion system: peak at maximum inspiration, trough and maximum expiration

It might be that your system does not conform to either of the two, and that the algorithms would have to be adapted. We'd be happy to help, but do not have the resources to test/validate.
